### ISSUE TITLE
DVR Functionality has stopped working and the video keeps looping 10sec clips.

### ISSUE THREAD

**Issue by tfabris** [tfabris/CrowCam#22]
Something has gone completely wrong. Attempting to watch the live feed results in a video that keeps going black and looping and showing only a 10 second clip before another bloop blackout.

DVR functionality is completely broken.

Possibly related to issue #20 but not sure.

**Comment by tfabris** [comment]
This is unrelated to CrowCamKeepAlive or Youtube-dl. In fact it is completely unrelated to my scripts at all. I shut down all of the scripts and the problem kept happening.

I have tried so far:
- Tried running CrowCamKeepAlive in Debug mode on the Synology. It is functioning as designed and it downloads the index file same as before. I also tried downloading and playing one of the .TS files linked in the index file, and that also played back normally. So everything is working as expected.
- Tried shutting down all script activity on the Synology. Problem still exists. In theory, I should be in "Plain" mode now, with my script doing nothing and just the Surveillance station providing the YouTube feed with no alterations or unusual behaviors. This was still exhibiting the problem. And that includes me actually watching the stream on YouTube while I'm doing it, so that alone should have "kept alive" the DVR functionality, and it didn't.
- Tried rebooting the camera itself. Did not fix the problem.
- Tried bouncing the Surveillance Station Live Stream feature by hand (with my scripts turned off). Did not fix the problem.
